packages/neon_framework/lib/src/widgets/dialog.dart

```diff
@@ -523,3 +555,395 @@ class NeonUnifiedPushDialog extends StatelessWidget {
         ],
       );
 }
+
+/// Shows an emoji picker.
+///
+/// When the user selects an emoji the dialog will pop and return the emoji.
+class NeonEmojiPickerDialog extends StatelessWidget {
+  /// Creates a new emoji picker dialog.
+  const NeonEmojiPickerDialog({
+    super.key,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    final theme = Theme.of(context);
+
+    return NeonDialog(
+      content: SizedBox(
+        width: NeonDialogTheme.of(context).constraints.maxWidth,
+        child: EmojiPicker(
```

Comment:
NeonDialog already does constrain the width for us (oversight in a prior rebase)

Reply:
Without setting the width we end up with `LayoutBuilder does not support returning intrinsic dimensions`.